src/pages/event/event.tsx

```diff
@@ -3,13 +3,27 @@ import React, { FunctionComponent, useState } from "react";
 import { isFuture, isPast } from "date-fns";
 import { Helmet } from "react-helmet";
 import { ResourceEvent, EventProps } from "../../components/UI/ResourceEvent";
-import { importAll } from "../../common/utils/importAll";
 import { Pagination, getPaginatedPosts, getPaginatedPagesTotal } from "@govtechsg/tradetrust-ui-components";
 import { getSortedByDateDesc } from "../../utils";
 import { Page } from "../../components/Layout/Page";
 
-let events = importAll(require.context("../../../cms/event/", false, /\.md$/)) as EventProps[];
-events = getSortedByDateDesc(events);
+// massage data by adding slug (base on filename generated by netlify cms)
+const getEvents = (context: __WebpackModuleApi.RequireContext) => {
+  const events: EventProps[] = [];
```

Comment:
i think this was used somewhere else too, consolidate to helper?

Reply:
ok